demo/paging/mock-server-results-service.ts

```diff
@@ -3,7 +3,7 @@ import {Observable} from "rxjs";
 import {PagedData} from "./model/paged-data";
 import {CorporateEmployee} from "./model/corporate-employee";
 import {Page} from "./model/page";
-var companyData = require('../../assets/data/company.json');
+var companyData = require('../../assets/data/forRowGrouping.json');
```

Comment:
At the moment I believe the code is still sharing this mock-server-results-service.ts, I need to created an exclusive one for the row-grouping so it doesn't touch the other demos. From this point onward  will perform the change before commenting on the next request.